modules/tinymce/src/plugins/table/main/ts/ui/CellDialog.ts

```diff
@@ -38,18 +38,34 @@ const getSelectedCells = (table: SugarElement<HTMLTableElement>, cells: SugarEle
   }));
 };
 
-const updateSimpleProps = (modifier: DomModifier, colModifier: DomModifier, data: CellData): void => {
-  modifier.setAttrib('scope', data.scope);
-  modifier.setAttrib('class', data.class);
-  modifier.setStyle('height', Utils.addPxSuffix(data.height));
-  colModifier.setStyle('width', Utils.addPxSuffix(data.width));
+const updateSimpleProps = (modifier: DomModifier, colModifier: DomModifier, data: CellData, shouldUse: (key: string) => boolean): void => {
+  if (shouldUse('scope')) {
+    modifier.setAttrib('scope', data.scope);
+  }
```

Comment:
I wish we could swap cell data to optional values instead of doing this, but that seems a little much for this fix.

Reply:
We'd still need to do this though, since "No data" is valid input here. We're checking if the data was changed, nothing else.